torch/csrc/jit/script/compiler.cpp

```diff
@@ -1254,10 +1248,10 @@ struct to_ir {
   std::shared_ptr<SugaredValue> emitApplyIdent(Ident ident, const std::vector<NamedValue>& inputs, at::ArrayRef<NamedValue> attributes, size_t n_binders) {
     auto it = function_table.find(ident.name());
     if (it != function_table.end()) {
-      return packOutputs(*graph, method.emit_call_to(ident.range(), it->second, inputs, attributes));
+      return std::make_shared<SimpleValue>(packOutputs(*graph, method.emit_call_to(ident.range(), it->second, inputs, attributes)));
     }
-    if(auto result = emitBuiltinCall(ident.range(), method, ident.name(), inputs, attributes, false)) {
-      return result;
+    if(auto result = emitBuiltinCall(ident.range(), *method.graph(), Symbol::aten(ident.name()), inputs, attributes, false)) {
+      return std::make_shared<SimpleValue>(result);
```

Comment:
Why do we assume the builtin calls return tensors now? What about e.g. `chunk`?

Reply:
I want to change Chunk to just return a list (which is a SimpleValue). Like `prim::FusedConcat` we can then turn into `prim::ConstantChunk` to maintain optimizations. 

For the API itself, it feels simpler to make builtins behave like Python, where there is a single return value, which might be a tuple.  Eventually we want to switch everything to be consistent:

1. builtins should just push an IValue tuple if the return value in the builtin was a tuple
2. python functions should push an IValue tuple for multiple returns. 
3. actual multiple returns should be reserved for `prim` ops like If/While and `prim::FusedConcat`

Given this future, it seems ok to make builtin just return a Value*